views/js/securesubmit.js

```diff
@@ -15,54 +19,170 @@
 
   function bindHandler() {
     var button = $(".ps-shown-by-js button");
-    button.off("click").on("click", secureSubmitFormHandler);
+    button.off("click").on("click", secureSubmitPrepareFields);
   }
 
-  function secureSubmitFormHandler(event) {
-    event.preventDefault();
-    $("#securesubmit-ajax-loader").show();
-    if ($("input[data-module-name='secureSubmit']").is(':checked')) {
-      $(".ps-shown-by-js button").html("Pay Now");
-      if ($('input.securesubmitToken').size() === 0) {
-        var card = $('.securesubmit-card-number').val().replace(/\D/g, '');
-        var cvc = $('.securesubmit-card-cvc').val();
-        var month = '';
-        var year = '';
-
-        if (
-          $('.securesubmit-card-expiry') &&
-          $('.securesubmit-card-expiry').val()
-        ) {
-          var date = $('.securesubmit-card-expiry').val().split('/');
+  function secureSubmitPrepareFields(){
+    // Create globalpayment configurations
+    $('#conditions_to_approve[terms-and-conditions]').prop('checked', true);
+    $("#secure-payment-field").attr("disabled",true);
+    securesubmit_public_key_test = securesubmit_public_key;
 
-          if (date[0]) {
-            month = date[0].trim();
-          }
+    // GlobalPayments JS enabled payment fields
+    GlobalPayments.configure({
+      publicApiKey: securesubmit_public_key_test
```

Comment:
Will the configured production API key work here too?

Reply:
@MSmedal this was the reference variable but still to avoid confusion I will rename the variable name. Thank you for suggestion.